### ISSUE TITLE
Calendar Broken as of Dart 1.12.1

### ISSUE THREAD

**Issue by janvladimirmostert** [rikulo/bootjack-datepicker#2]
Bootjack Datepicker / Calendar stopped working after a Dart upgrade.

```
Calendar.use();
Calendar.wire(querySelector("div.calendar"));
```

It's failing in calendar.dart on line 325:

```
  void _dayView() {
    TableElement calBody = element.querySelector('.cnt');
    Element dow = calBody.tBodies[0].createFragment(_DOW_TEMPLATE).children[0];
    List<Element> children = dow.children;
    
    List<String> swkDays = _dfmt.dateSymbols.SHORTWEEKDAYS;
    int ofs = (_firstDayOfWeek + 1) % 7;
    //render week days
    for (int i = swkDays.length; --i >= 0;) {
      children[i].text = swkDays[(i + ofs) % 7];
    }
    
    var buffer = new StringBuffer();
    for (int i = 6; --i >= 0;) {
      buffer.write(_DAYROW_TEMPLATE);
    }
    
    calBody.tBodies[0]
    ..append(dow)
    ..appendHtml(buffer.toString(), treeSanitizer: NodeTreeSanitizer.trusted);   <<<<<<<< ERROR
  }
```

The exception is:

> Exception: No static getter 'trusted' declared in class 'NodeTreeSanitizer'.
> NoSuchMethodError: method not found: 'trusted'
> Receiver: Type: class 'NodeTreeSanitizer'

Posted on StackOverflow in the meantime:
https://stackoverflow.com/questions/32836774/dart-treesanitzer-no-static-getter-trusted-declared-in-class-nodetreesanitiz

**Comment by tomyeh** [comment]
Unable to reproduce it.

NodeTreeSanitizer.trusted is available only on 1.12 or later. Please make sure your Dartium is running on the right version. Please reopen if it is not the case.

**Comment by janvladimirmostert** [comment]
Yes, it's indeed due to an older version of Chromium which didn't upgrade due Homebrew merging the two packages, was just about to close it myself.
